src/components/breeding.html

```diff
@@ -158,10 +160,6 @@ <h5 class="modal-title" id="breedingModalLabel" data-bind="tooltip: { title: `Yo
                         </ul>
                         <!-- /ko -->
                     </div>
```

Comment:
Looks like you missed a `</div>` (pair of the one from the old line 146), and it is causing the close button to wrap to the right of the modal.

Old:
![image](https://user-images.githubusercontent.com/90011/95086264-83943780-06dd-11eb-83c7-fec587b1fc29.png)

Now:
![image](https://user-images.githubusercontent.com/90011/95086360-a3c3f680-06dd-11eb-86e5-6b0226da7632.png)

While you're fixing that, please also unindent that block by one level (4 spaces) so that we have:
```
                <div class="tab-pane" id="breeding-fossils">
                    <p class="text-danger" data-bind="visible: !App.game.breeding.hasFreeEggSlot()">
```

instead of
```
                <div class="tab-pane" id="breeding-fossils">
                        <p class="text-danger" data-bind="visible: !App.game.breeding.hasFreeEggSlot()">
```